about.html

```diff
@@ -0,0 +1,174 @@
+						<span class="navbar-toggler-icon"></span>
+					</button>
+					<div class="collapse navbar-collapse" id="navbarNav">
+						<ul class="navbar-nav">
+							<li class="nav-item">
+								<img src="assets\logos\icons8-paid-26.png" class="cart" alt="" width="26px" height="30px">
+								<a class="nav-link activ" aria-current="page" href="#"><b class="cartText">Cart</b></a>
+							</li>
+							<li class="nav-item">
+								<a class="nav-link" href="#">Sign up</a>
+							</li>
+							<li class="nav-item" id="yo">
+								<a class="nav-link" href="#">Sign in</a>
+							</li>
+							<!-- <li class="nav-item"> -->
+							<!-- <a class="nav-link disabled"></a> -->
+							<!-- </li> -->
+						</ul>
+					</div>
+				</div>
+			</nav>
+		</div>
+	</nav>
+	<!-- Navbar Completed -->
+	<br>
+	<br>
+
+	<style>
+		* {
+			box-sizing: border-box;
+			/* background-color: black; */
+		}
+
+		body {
+			font-family: Verdana, sans-serif;
+			/* color: black; */
+			background-color: black;
+		}
+
+        #about-title {
+            text-align: center;
+            padding: 0;
+            color: rgb(141, 140, 140);
+	        font-family: 'Playfair Display SC', serif;
+            letter-spacing: .10em;
+        }
+
+		#about-footer {
+			position: absolute;
+			bottom: 0;
+			width: 100%;
+			text-align: center;
+		}
+
+		.footer {
+			text-align: left;
+			justify-content: center;
+			margin: auto;
+			max-width: 1000px;
+		}
+
+		/* On smaller screens, decrease text size */
+		@media only screen and (max-width: 300px) {
+			.text {
+				font-size: 11px
+			}
+		}
+
+	</style>
+
+<div>
+    <h3 id="about-title">ABOUT US</h3>
+</div>
+<br>
+<br>
+
+</body>
+<br>
+<br>
+	<Footer id="about-footer">
+		<div class="footer resp ">
+			<div class="About">
+				<div class="ABOUT  ">ABOUT</div>
+				<a href="#" id="foot">Contact</a>
+				<a href="./about.html" id="foot">About</a>
+				<a href="#" id="foot">Press</a>
+			</div>
+			<div class="About">
+				<div class="ABOUT ">HELP</div>
+				<a href="#" id="foot">Payment</a>
+				<a href="#" id="foot">Shipping</a>
+				<a href="#" id="foot">FAQ</a>
+			</div>
+			<div class="About">
+				<div class="ABOUT ">POLICY</div>
+				<a href="#" id="foot">Security</a>
+				<a href="#" id="foot">Privacy</a>
+				<a href="#" id="foot">Return Policy</a>
+			</div>
+			<div class="About resp2">
+				<div class="ABOUT ">SOCIAL</div>
+				<a href="#" id="foot">Instagram</a>
+				<a href="#" id="foot">Twitter</a>
+				<a href="#" id="foot">Youtube</a>
+			</div>
+			<div class="About">
+				<div class="ABOUT">Mail Us:</div>
+				<p id="mail">D&W Internet Private Limited,
+
+					Buildings Alyssa, Begonia &
+
+					Clove Embassy Tech Village,
+
+					Outer Ring Road, Devarabeesanahalli Village,
+
+					Bengaluru, 560103,
+
+					Karnataka, India</p>
+			</div>
+		</div>
+
+		<p id="footborder" style="max-width: 1000px;"></p>
```

Comment:
Align this footborder to center

Reply:
I could not for the life of me figure out how to align that footer. But I'll add responsiveness and comments I forgot

Reply:
Great go ahead!